src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkSectionRedeclarationInspection.kt

```diff
@@ -33,12 +36,20 @@ class SmkSectionRedeclarationInspection : SnakemakeInspection() {
                     registerProblem(
                             it,
                             SnakemakeBundle.message("INSP.NAME.section.redeclaration.message", name),
-                            ProblemHighlightType.LIKE_UNUSED_SYMBOL
+                            ProblemHighlightType.LIKE_UNUSED_SYMBOL,
+                            null, RemoveSectionQuickFix()
```

Comment:
Maybe also change section name (w/o updating usages)? E.g.
```
rule foo:
   output: "file.out"
   output: "file.out.log"
```
to 
```
rule foo:
   output: "file.out"
   log: "file.out.log"
```
Such duplication could be a result of copy-paste  + user forgot to update section name

Reply:
done